homeassistant/components/home_connect/button.py

```diff
@@ -94,15 +94,9 @@ def __init__(
         super().__init__(
             coordinator,
             appliance,
-            # The entity is subscribed to the appliance connected event,
-            # but it will receive also the disconnected event
-            ButtonEntityDescription(
-                key=EventKey.BSH_COMMON_APPLIANCE_CONNECTED,
-            ),
+            desc,
+            (appliance.info.ha_id,),
```

Comment:
It looks like the context isn't the same anymore (there's no event key included). Is that indended?

Reply:
Yes, we only need to have haId at context to be aware of the general events (such CONNECTED and DISCONNECTED)

Reply:
Ok.